src/gui/src/clockWidget.cpp

```diff
@@ -1544,4 +1615,51 @@ void ClockWidget::fit()
   }
 }
 
+void ClockWidget::findInCts(const Selected& selection)
+{
+  if (!selection) {
+    return;
+  }
+  findInCts(SelectionSet({
+      selection,
+  }));
+}
+
+void ClockWidget::findInCts(const SelectionSet& selections)
+{
+  if (views_.empty()) {
+    return;
+  }
+  if (selections.empty()) {
+    return;
+  }
+
+  std::set<int> found;
+  std::set<int> not_found;
```

Comment:
These could have better names or at least a comment to make it clear that they're sets of indices of the views in which there are/aren't selected instances.

Reply:
Fixed.

Changed `found` to `changed_views` and `not_found` to `not_changed_views`.
Also removed the variable `min_found`, now if the current view is not in the `changed_views` set, the current view is changed to `*(changed_views.begin())`.